frontend/src/modules/dashboard/pages/dashboard-page.vue

```diff
@@ -35,6 +35,9 @@
           </div>
         </div>
       </div>
+      <div class="mt-auto mb-6">
+        <lfx-footer class="px-2 max-w-3xl" />
+      </div>
```

Comment:
Is it supposed not to be centered here?
Also, dashboard also uses layout I believe. Couldn't we re-use the same footer in all pages?
![Screenshot 2024-12-04 at 08 59 03](https://github.com/user-attachments/assets/2d49cc0d-36a6-4b2e-94a1-86c08aceee41)

Reply:
yeah, that's why we have 2 footer one in layout and one in dashboard

Reply:
fixed